controllers/cluster/cluster_controller.go

```diff
@@ -90,87 +88,27 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 
 	log = log.WithValues("Cluster", cluster.Namespace+"/"+cluster.Name)
 
-	if _, exist := cluster.Labels[akoov1alpha1.AviClusterLabel]; !exist {
+	adcName, exist := cluster.Labels[akoov1alpha1.AviClusterLabel]
```

Comment:
akodeploymentconfig reconciler will add this `adcName:avi` label to the cluster once the adc matches?

Reply:
Kind of, it will be added in the mapper function here: https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/blob/3c7e4f845dab3073be75a5f188df48ef5db503aa/pkg/handlers/cluster_for_akodeploymentconfig_handler.go#L38